lib/benchee/statistics.ex

```diff
@@ -160,7 +167,8 @@ defmodule Benchee.Statistics do
     deviation           = standard_deviation(run_times, average, iterations)
     standard_dev_ratio  = deviation / average
     standard_dev_ips    = ips * standard_dev_ratio
-    median              = Percentile.percentile(run_times, iterations, 50)
+    percentiles         = Percentile.percentile(run_times, [50, 99])
+    median              = Map.get(percentiles, 50)
```

Comment:
:+1:

Reply:
I prefer `Map.fetch!` though as I like to blow up early :)

Reply:
👍